app/modules/FrontModule/Registration/components/RegistrationForm/registrationForm.latte

```diff
@@ -120,6 +120,8 @@
 
 	<div class="row">
 		<div class="columns medium-offset-3">
+                        {input email}
```

Comment:
tohle by nemelo byt potreba... doplni NF sam ?

Reply:
Mám zkušenost, že občas při manuálním vykreslování to NF nedoplní a tedy poté nastávají problémy. Proto jsem se naučil to všude explicitně uvádět.